qemu/qemu.go

```diff
@@ -2402,8 +2402,12 @@ type Knobs struct {
 	Realtime bool
 
 	// Exit instead of rebooting
+	// Prevents QEMU from rebooting in the event of a Triple Fault.
```

Comment:
This is not exactly related to this commit.

Reply:
Yes. This just ensure that the comment is accurate. I will amend the commit message to reflect it.

Reply:
Thanks!